### PR TITLE
Adjust description when plugin failed to be copied to match button name

### PR DESCRIPTION
**Pull request by TheMagnetar**
**When merged this pull request will:**
- Title

image (5)

### CODE REVIEW

File: extensions/src/ACRE2Steam/CallExt_DllMain.cpp [hunk 1]
```diff
@@ -410,7 +410,7 @@ void __stdcall RVExtension(char *output, int outputSize, const char *function) {
                             if (last_error != ERROR_FILE_NOT_FOUND) {
                                 updateRequired = true;
                                 if (last_error == FILE_SHARE_DELETE) { // File in use
-                                    const int32_t result = MessageBoxA(nullptr, "ACRE2 is unable to copy the TeamSpeak 3 plugin due to it being in use. Please close any instances of TeamSpeak 3 and click Retry.\n\nIf you would like to close Arma 3, click Cancel. Press Continue to launch Arma 3 regardless", "ACRE2 Installation Error", MB_CANCELTRYCONTINUE | MB_ICONEXCLAMATION);
+                                    const int32_t result = MessageBoxA(nullptr, "ACRE2 is unable to copy the TeamSpeak 3 plugin due to it being in use. Please close any instances of TeamSpeak 3 and click \"Try Again\".\n\nIf you would like to close Arma 3, click Cancel. Press Continue to launch Arma 3 regardless", "ACRE2 Installation Error", MB_CANCELTRYCONTINUE | MB_ICONEXCLAMATION);
```
Comment (bux) [inline extensions/src/ACRE2Steam/CallExt_DllMain.cpp]:
```suggestion
                                    const int32_t result = MessageBoxA(nullptr, "ACRE2 is unable to copy the TeamSpeak 3 plugin due to it being in use. Please close any instances of TeamSpeak 3 and click \"Try Again\".\n\nIf you would like to close Arma 3, click Cancel. Press Continue to launch Arma 3 regardless.", "ACRE2 Installation Error", MB_CANCELTRYCONTINUE | MB_ICONEXCLAMATION);
```

File: extensions/src/ACRE2Steam/CallExt_DllMain.cpp [hunk 2]
```diff
@@ -360,7 +360,7 @@ void __stdcall RVExtension(char *output, int outputSize, const char *function) {
                         if (!CopyFileA((LPCSTR)path_x86.c_str(), (LPCSTR)ts_path_x86.c_str(), false) || !CopyFileA((LPCSTR)path_x64.c_str(), (LPCSTR)ts_path_x64.c_str(), false)) {
                             DWORD last_error = GetLastError();
                             if (last_error == 32) {
-                                const int32_t result = MessageBoxA(nullptr, "ACRE2 was unable to copy the TeamSpeak 3 plugin due to it being in use. Please close any instances of TeamSpeak 3 and click Retry.\n\nIf you would like to close Arma 3, click Cancel. Press Continue to launch Arma 3 regardless", "ACRE2 Installation Error", MB_CANCELTRYCONTINUE | MB_ICONEXCLAMATION);
+                                const int32_t result = MessageBoxA(nullptr, "ACRE2 was unable to copy the TeamSpeak 3 plugin due to it being in use. Please close any instances of TeamSpeak 3 and click \"Try Again\".\n\nIf you would like to close Arma 3, click Cancel. Press Continue to launch Arma 3 regardless", "ACRE2 Installation Error", MB_CANCELTRYCONTINUE | MB_ICONEXCLAMATION);
```
Comment (bux) [inline extensions/src/ACRE2Steam/CallExt_DllMain.cpp]:
```suggestion
                                const int32_t result = MessageBoxA(nullptr, "ACRE2 was unable to copy the TeamSpeak 3 plugin due to it being in use. Please close any instances of TeamSpeak 3 and click \"Try Again\".\n\nIf you would like to close Arma 3, click Cancel. Press Continue to launch Arma 3 regardless.", "ACRE2 Installation Error", MB_CANCELTRYCONTINUE | MB_ICONEXCLAMATION);
```

Comment (bux) [inline extensions/src/ACRE2Steam/CallExt_DllMain.cpp]: Add period after `regardless`

File: extensions/src/ACRE2Steam/CallExt_DllMain.cpp (hunk 1 above)
Comment (bux) [inline extensions/src/ACRE2Steam/CallExt_DllMain.cpp]: Add period after `regardless`

**Comment by bux** [comment]
image

Typo?

`,` -> `.`

**Review by bux — APPROVED** [review]

**Review by jonpas — APPROVED** [review]

**Pull request merged**